config/jdbc/chipsDS-jdbc.xml

```diff
@@ -21,7 +21,7 @@
     <initial-capacity>1</initial-capacity>
     <max-capacity>25</max-capacity>
     <capacity-increment>1</capacity-increment>
-    <shrink-frequency-seconds>300</shrink-frequency-seconds>
+    <shrink-frequency-seconds>0</shrink-frequency-seconds>
```

Comment:
is there no shrinking-enabled=false option ?

Reply:
Per doco:

When set to 0, shrinking is disabled.

Reply:
yes, it's right. Config is handy when we want to update the value. Thanks.